rust/chains/hyperlane-cosmos/src/providers/grpc.rs

```diff
@@ -271,6 +285,59 @@ impl WasmGrpcProvider {
         .map_err(Into::<HyperlaneCosmosError>::into)?;
         Ok(account)
     }
+
+    /// Injective-specific logic for querying an account.
+    async fn account_query_injective(&self, account: String) -> ChainResult<BaseAccount> {
+        let request = tonic::Request::new(
+            injective_std::types::cosmos::auth::v1beta1::QueryAccountRequest { address: account },
```

Comment:
if I were to nitpick I'd say we should feature flag `account_query_injective` and the injective-related dependencies + imports but it's no big deal. Maybe worth adding a comment about this though

Reply:
hmm wouldn't this mean we'd just always have that feature flag enabled, and we'd still have a runtime check? because we may wanna relay between Neutron and Injective, so we need both injective and non-injective code present in this crate and a runtime check to use injective when appropriate. And I'm not sure we'd want to ship a different binary depending on whether we're using injective or not